Makefile

```diff
@@ -99,15 +101,23 @@ dev-setup:
 	@echo "------------------"
 
 run-x86:
-	@docker-compose -f ./deploy/docker/clickhouse-setup/docker-compose.yaml up -d
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.yaml up -d
 
 run-arm:
-	@docker-compose -f ./deploy/docker/clickhouse-setup/docker-compose.arm.yaml up -d
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.arm.yaml up -d
+
+down-x86:
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.yaml down -v
+
+down-arm:
+	@docker-compose -f $(STANDALONE_DIRECTORY)/docker-compose.arm.yaml down -v
 
 clear-standalone-data:
-	@docker run --rm -v "$(PWD)/deploy/docker/clickhouse-setup:/pwd" busybox \
-	sh -c "cd /pwd/data && rm -rf alertmanager/* clickhouse/* signoz/*"
+	@cd $(SWARM_DIRECTORY)
```

Comment:
@prashant-shahi should this be `STANDALONE_DIRECTORY`?

Reply:
That was updated a while back.